ci/_environment.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-ANDROID_BUILDER_TAG=28e6bacd68
+ANDROID_BUILDER_TAG=eddf45adfa
```

Comment:
What has been changed?

Reply:
https://github.com/avito-tech/avito-android/pull/813
it is changes from that pr